### ISSUE TITLE
Verify token provided by Keycloak to allow access to server api

### ISSUE THREAD

**Issue by petmongrels** [avniproject/avni-server#395]
For testing use postman and get a token from Keycloak and call Avni's protected REST API.

**Comment by petmongrels** [comment]
code is working locally....blocked by automation/deployment story

**Comment by himeshr** [comment]
Task List:
=========
*) Docker image //DONE
*) realm file //DONE
*) postman api , amrit token //DONE
*) Generate token using username and PWD //DONE
*) IMPORTANT: Modified client scope to include custom:userUUID in all token content //DONE
*) Created admin-api client with client-secret to be used while connecting from avni-server to keycloak-server //DONE
*) Generate token using client_secret //DONE
*) BaseIAMService.getUserFromToken() //done
*) BaseIAMService is extended by cog and key //done


*) Use profile Staging or on-premise //TODO
*) IDP, use userContext to determine storageService bean //TODO
*) CognitoIDP service alternative for keycloak, Not yet wired KeycloakIDPService as substitute for CognitoIdpService. //TODO
*) Write test cases for Cognito using Postman //TODO
*) Write test cases for keycloakIDP using Postman //TODO
*) Automate Keycloak deployment with service creation //POSTPONED

**Comment by himeshr** [comment]
Pending tasks:
===========
*) Write test cases for keycloakIDP using Postman //TODO
*) Automate Keycloak deployment with service creation //POSTPONED
